gamingcardano

```diff
@@ -0,0 +1,15 @@
+  {
```

Comment:
You need to add these changes to the registry.json file. Have a look at the other pull requests if you are not sure what this means